code/datums/trading/trade.dm

```diff
@@ -32,7 +32,7 @@
 	what_want			What the person says when they are asked if they want something
 
 	*/
-	var/want_multiplier = 2                                     //How much wanted items are multiplied by when traded for
+	var/want_multiplier = 5                                     //How much wanted items are multiplied by when traded for
```

Comment:
The default want multiplier of 2 is fine.

Reply:
For the initial run let's go with keeping the multiplier normal and normal prices - maybe the reason why so many merchants failed was simply due to amount of garbage in the pools that now was adjusted. Comitting your suggestions & editing changelog, then resolving.

Reply:
commit pushed, resolving.